plugin-server/src/main/pluginsServer.ts

```diff
@@ -17,7 +17,6 @@ import { Hub, PluginServerCapabilities, PluginsServerConfig } from '../types'
 import { createHub, createKafkaClient, createKafkaProducerWrapper } from '../utils/db/hub'
 import { PostgresRouter } from '../utils/db/postgres'
 import { cancelAllScheduledJobs } from '../utils/node-schedule'
-import { PeriodicTask } from '../utils/periodic-task'
```

Comment:
Also, I left this alone because it does seem like it _could_ be useful later… but since it's no longer used, it might be better to just get rid of it than expect anybody to remember it exists later if they want to do something similar since we'll more than likely end up with multiple implementations of the same idea

Reply:
🤷‍♂️ 

Seems relatively harmless to leave it around, either way.